.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,31 @@
+name: Test
+
+on: pull_request
+
+env:
+  GOPROXY: https://proxy.golang.org
+
+jobs:
+  test-harbor:
+    name: Test on Harbor
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        harbor: [v1.10.0, v1.10.1, v1.10.2]
+        go: [1.13]
```

Comment:
Seems we did have builds with Go 1.14 at some time (since we have required status checks for Go 1.14). Any reason why we're only testing against 1.13 now?

Reply:
Since we've started writing this client in Go 1.13, we decided to drop the 1.14 (and 1.12) matrix tests to be sure that Github Actions pass and don't fail because of compatibility reasons.

Reply:
OK